clearbit/clearbit.go

```diff
@@ -21,8 +22,9 @@ type Client struct {
 // Config represents all the parameters available to configure a Clearbit
 // client
 type Config struct {
-	apiKey     string
-	httpClient *http.Client
+	apiKey         string
+	httpClient     *http.Client
+	secondsTimeout int
```

Comment:
So it should work by doing `clearbit.NewClient(clearbit.WithTimeout(20 * time.Second))` right?